.github/test_plan.md

```diff
@@ -68,6 +68,9 @@
     1. Verify the output from what you added
 -   [ ] Verify dark and light main themes
     1. Repeat the `Start and connect to local server` and `Verify basic outputs` steps using `Default Dark+` and `Default Light+` themes
+-   [ ] Verify tooltips in Python file that has been run in the interactive window
+    1. After manualTestFile.py has been run in the interactive window, place your cursor over the `myTensor` variable.
+    1. Verify that the tooltip that appears contains four lines where the first line says `myTensor: Tensor`, the second line has `shape` information, the third line has `dtype` information, and the fourth line has `device` information.
```

Comment:
Do we need this if we have an end to end test for this.
I think we should only add stuff that's difficult to test via code

Reply:
Aren't we supposed to update the test plan with new functionality? I'm happy to remove this, as the test I have simulates this functionality pretty well. Just not sure what our policy on test plan updates is.

Reply:
I don't think it hurts anything here other than it taking longer for CTI to test.